src/renderer/remote.ts

```diff
@@ -249,6 +249,7 @@ function proxyFunctionProperties (remoteMemberFunction: Function, metaId: number
 
 // Convert meta data from browser into real value.
 function metaToValue (meta: MetaType): any {
+  if(!meta) return {}
```

Comment:
```suggestion
  if (!meta) return {}
```

CI doesn't catch lint issues it seems?